projects/step-frontend/src/lib/modules/timeseries/modules/filter-bar/pipes/visible-filter-item.pipe.ts

```diff
@@ -1,8 +1,9 @@
 import { Pipe, PipeTransform } from '@angular/core';
-import { FilterBarItem } from '../performance-view/filter-bar/model/filter-bar-item';
+import { FilterBarItem } from '../../_common';
 
 @Pipe({
   name: 'visibleFilterItems',
+  standalone: true,
```

Comment:
All modules in projects have approximately the following structure:

- components - folder for components
- directives - folder for directives
- pipes - folder for pipes
- injectables (or services) - folder for services and injection tokens
- types (or shared) - additional interfaces, enums and other types
- modules - folder for sub modules

Restructuring whole time-series module into flat structure, would make it hard to support, that's why I split it in several sub modules:
- _common
- chart
- filter-bar
- legacy 

Since angular 14, defining module object became not necessary. And for this four sub-modules it could be only used for declaring and exporting components only. So instead of defining module's object I make it's entities as standalone and export only required parts with `index.ts` files in each sub module.